extensions/sql-migration/src/api/azure.ts

```diff
@@ -640,8 +816,100 @@ export interface StartDatabaseMigrationRequest {
 
 export interface StartDatabaseMigrationResponse {
 	status: number,
-	databaseMigration: DatabaseMigration
-	asyncUrl: string
+	databaseMigration: DatabaseMigration,
+	asyncUrl: string,
+}
+
+export enum Kind {
+	SQLDB = 'SqlDb',
+	SQLMI = 'SqlMi',
+	SQLVM = 'SqlVm',
+}
+
+export interface validateIrSqlDatabaseMigraionRequest {
```

Comment:
typo in interface name: **V**alidateIrSqlDatabase**Migration**Request

(capitalize first letter, and misspelled "Migration")